.github/workflows/ci.yml

```diff
@@ -12,20 +12,23 @@ jobs:
       fail-fast: false
       matrix:
         command:
-          - "++2.11.12 testsJVM/test"
-          - "++2.12.10 testsJVM/test"
-          - "++2.12.11 testsJVM/test"
-          - "++2.13.1 testsJVM/test"
-          - "download-scala-library ++2.12.11 testsJVM/slow:test"
-          - "++2.11.12 testsJS/test"
-          - "++2.12.10 testsJS/test"
-          - "++2.13.1 testsJS/test"
-          - "mima"
+          - "sbt ++2.11.12 testsJVM/test"
+          - "sbt ++2.12.10 testsJVM/test"
+          - "sbt ++2.12.11 testsJVM/test"
+          - "sbt ++2.13.1 testsJVM/test"
+          - "sbt download-scala-library ++2.12.11 testsJVM/slow:test"
+          - "SCALAJS_VERSION=0.6.32 sbt ++2.11.12 testsJS/test"
```

Comment:
Let's drop 0.6.x support, users on 0.6.x can continue to use older Scalameta versions. Based on my experience in MUnit, it significantly increases the complexity of the build and release process when supporting Scala.js 0.6.x and 1.x